### ISSUE TITLE
Make Bitwarden integration optional (test branch)

### ISSUE THREAD

**Issue by Joery** [pivpn/pivpn#819]
## In raising this issue, I confirm the following: 

- [X] I have read and understood the [contributors guide](https://github.com/pivpn/pivpn/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting can be *is* directly related to the pivpn installer script.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pivpn/pivpn/wiki/FAQ), [closed issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed), and [open issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen)).

### Issue

More of a suggestion and not an issue, it would be great if the Bitwarden integration was optional. Having PiVPN install Node.js, npm & Bitwarden kind of defeats the purpose for me for PiVPN being a shell script to quickly install a secure VPN as I'm not using Bitwarden myself. (Among many others)

I think a simple Yes / No screen during the install would be sufficient for most.

See https://github.com/pivpn/pivpn/commit/b60a06791d755ef21bc586cbf702ed653615a068, right now it's only in the testing branch

**Comment by 4s3ti** [comment]
Hi, @Joery Test branch has just been merged into master, I can understand your point! 
Bit warden is supported however it won't prompt anymore if you want to add the ovpn as a secure note.
I have removed that part of the code as during my tests today I have noticed it can break the ovpn creation. 

I will mark this as an enhancement, to somehow make bitwarden optional.

**Comment by orazioedoardo** [comment]
I agree, I think the script should just detect an existing bitwarden installation and offer to save the client to bitwarden. Installing it seems overkill to me.

**Comment by 4s3ti** [comment]
working on getting a whiptail message asking if user wants to install it and integrating it into pivpndebug, also adding some error handling. 

should be pushed during today.

**Comment by 4s3ti** [comment]
@Joery pushed insto master with commit 50ad223e83fb96d20d732d9c35dedd2f0186dadc
check if its ok, and comeback with feedback pls.

**Comment by 4s3ti** [comment]
Turns out that bitwarden might cause errors depending on each user environment, having this said and according to the conversation here: https://github.com/pivpn/pivpn/issues/823 I have removed bitwarden installation from the script with this commit: 510979ae90682c4ca11eb59d763c91930ffb5847 keeping the integration only with pivpn add ... users wanting to use bitwarden should install it themselves manually. 

Thanks a lot for your feedback and contribution! 
Closing.
